### ISSUE TITLE
Card `selectable` state - Ensure custom controls provide proper textual name, role, and state information - (2036570012) 

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5585]
### Module: 
 18b Card 

### Violation: 
 Ensure custom controls provide proper textual name, role, and state information 

![image](https://user-images.githubusercontent.com/5023024/202752959-f25fc7d0-fef4-4c60-84ff-e81dafc5667e.png)

### WCAG Reference: 
- [1.1.1 Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)
- [4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html) 

### Instance ID: 
 2036570012 

### Severity: 
 7 

### Description: 
 [Issue]
There are checkboxes present under the following sections without appropriate name information. Currently the checkboxes have aria-label as "select/deselect" which in inappropriate. Examples include:

- Test 3. Selected Card
- Test 2. Selectable Card

[User Impact]
Screen reader users will be unable to determine that these controls can be checked or unchecked. They will also be unable to review which options they have chosen.

[Code Reference] 
 ```
<div aria-checked="false" aria-label="Select" class="toggle" role="checkbox" tabindex="0"><svg class="check-svg" viewBox="0 0 16 16"><path d=""></path></svg><slot></slot></div>
 ``` 

### Note: 
 [Suggestion]
Ensure custom controls provide proper textual name, role, and state information.

The best way to do this is to use native controls, as they come with this information built in. In this case, ``` <input type="checkbox"> ``` elements would be most appropriate.

Developers can often achieve the same visual appearance using progressively enhanced CSS. Alternatively, they can visually hide the checkboxes with opacity: 0 and use SVGs to fake their appearance.

If conversion to native checkboxes is not possible, the following information is expected:
- The element functioning as the checkbox must have role="checkbox".
- This checkbox element must have a label provided by internal text, aria-label, or aria-labelledby.
- When checked, the checkbox element must have aria-checked="true", and when not checked, it must have aria-checked="false". If partially checked, it must have aria-checked="mixed".
- Checkboxes in a logical group must be children of an element with role="group". That element must have aria-labelledby set to the ID of text labelling the group.

[Compliant Code Example] 
 ```
<div aria-checked="false" aria-label="Selectable card title 1" class="toggle" role="checkbox" tabindex="0"><svg class="check-svg" viewBox="0 0 16 16"><path d=""></path></svg><slot></slot></div>
 ```
 

### Media Type: 
 Custom Controls 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
Needs research to determine if `checked` is being accurately depicted in the components role.

**Comment by geospatialem** [comment]
Verified on `master` with NVDA and JAWS.
